src/HttpTester.js

```diff
@@ -0,0 +1,57 @@
+const Metrics = require('metrics-sharelatex')
+const Settings = require('settings-sharelatex')
+const Logger = require('logger-sharelatex')
+
+const request = require('request-promise-native')
+
+const PodLocator = require('./PodLocator')
+
+module.exports = class {
+  constructor() {
+    this.podLocator = new PodLocator()
+  }
+
+  async run() {
+    await Promise.all(
+      // run test on all sibling pods
+      this.podLocator.siblings
+        .map(async host => {
+          // run all settings-defined tests
+          Settings.tests.http.map(async test => {
+            const timer = new Metrics.Timer(
+              `debugger.http.${test.name}.${this.podLocator.self}.${host}`
```

Comment:
I suspect the metric should be called `debugger.http` and the other values should be labels?

Reply:
Yes, quite possibly. Thanks :+1: